examples/README.md

```diff
@@ -149,3 +153,80 @@ This time we are not calling subscription call, but they are automatically creat
 
 - Fill `john.doe` (this is set in JWT) to first input and some message to second input => update should be displayed below after few moments
 - Fill any other name to first input => this time no update should arrive
+
+# Longpolling
```

Comment:
Would be good to include examples as soon as possible -- to properly test all important cases.

Reply:
yes - unfortunately i didn't find time yet to do so...